build.sbt

```diff
@@ -1,110 +1,99 @@
-import com.typesafe.sbt.SbtScalariform.ScalariformKeys
 import scalariform.formatter.preferences._
 import ReleaseTransformations._
 
 enablePlugins(CrossPerProjectPlugin)
 
 lazy val core = (project in file("core")).
-  settings(settings,
-    tutTargetDirectory := file(".")
-  )
+  enablePlugins(TutPlugin).
+  settings(commonSettings, tutTargetDirectory := file("."))
 
 lazy val docs = (project in file("docs")).
-  settings(settings, publishArtifact := false).
+  enablePlugins(TutPlugin).
+  settings(commonSettings, publishArtifact := false).
   dependsOn(core)
 
-lazy val cats = (project in file("modules/cats")).
-  settings(settings).
-  dependsOn(core)
+def module(proj: Project) = proj.
+  enablePlugins(TutPlugin).
+  dependsOn(core).
+  dependsOn(core % "test->test"). // In order to reuse the scalacheck generators
```

Comment:
Yes but we already discussed of sharing the testing part of PureConfig. Before there was no need but now we may want to have all modules to use the same format for tests as `core`.